### PR TITLE
Laserpointer - Block laser attachments if setting disabled (CBA 3.16.0)

### PR DESCRIPTION
**Pull request by PabstMirror**
req https://github.com/CBATeam/CBA_A3/pull/1595

### CODE REVIEW

File: addons/laserpointer/XEH_postInit.sqf [hunk 1]
```diff
@@ -15,12 +15,14 @@ GVAR(greenLaserUnits) = [];
 ["CBA_settingsInitialized", {
     // If not enabled, dont't add draw eventhandler or PFEH (for performance)
     if (!GVAR(enabled)) exitWith {
+        ["ACE_acc_pointer_red", { false }] call CBA_fnc_addAttachementCondition;
+        ["ACE_acc_pointer_green", { false }] call CBA_fnc_addAttachementCondition;
```
Comment (jonpas) [inline addons/laserpointer/XEH_postInit.sqf:19]:
```suggestion
        ["ACE_acc_pointer_red", { false }] call CBA_fnc_addAttachmentCondition;
        ["ACE_acc_pointer_green", { false }] call CBA_fnc_addAttachmentCondition;
```

File: addons/main/script_mod.hpp [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 // MINIMAL required version for the Mod. Components can specify others..
 #define REQUIRED_VERSION 2.12
-#define REQUIRED_CBA_VERSION {3,15,7}
+#define REQUIRED_CBA_VERSION {3,15,9}
```
Comment (jonpas) [inline addons/main/script_mod.hpp:14]:
```suggestion
#define REQUIRED_CBA_VERSION {3,16,0}
```

**Review by jonpas — APPROVED** [review]

**Comment by jonpas** [comment]
Can merge this if we release CBA at the same time. Setting milestone so it's not forgotten.

File: addons/main/script_mod.hpp [hunk 3]
```diff
@@ -11,7 +11,7 @@
 
 // MINIMAL required version for the Mod. Components can specify others..
 #define REQUIRED_VERSION 2.12
-#define REQUIRED_CBA_VERSION {3,15,7}
+#define REQUIRED_CBA_VERSION {3,16,0}
```
Comment (PabstMirror) [inline addons/main/script_mod.hpp:14]:
```suggestion
#define REQUIRED_CBA_VERSION {3,15,7}
```

**Comment by PabstMirror** [comment]
can merge now, and just bump cba when we do ace versioning
`call nil` is safe

**Pull request merged**